index.html

```diff
@@ -16,6 +17,9 @@
       <a href="">
         <p class="welcome"> <strong>Welcome</strong></p>
       </a>
+      <a href="" class="desktop_menu">Portfolio</a>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure that when the user clicks either Portfolio or Contact it should take him to that particular section